src/cocotb_tools/runner.py

```diff
@@ -302,6 +302,7 @@ def test(
         gpi_interfaces: Optional[List[str]] = None,
         testcase: Optional[Union[str, Sequence[str]]] = None,
         seed: Optional[Union[str, int]] = None,
+        elab_args: Sequence[str] = [],
```

Comment:
You will though need to add documentation to the docstring below.

Reply:
Hi, I've added this in the last commits.